sympy/matrices/expressions/determinant.py

```diff
@@ -32,13 +33,13 @@ def __new__(cls, mat):
     def arg(self):
         return self.args[0]
 
-    def doit(self, expand=False):
+    def doit(self, expand=False, **kwargs):
         try:
-            return self.arg._eval_determinant()
+            return self.arg._eval_determinant(**kwargs)
         except (AttributeError, NotImplementedError):
             return self
 
-def det(matexpr):
+def det(matexpr, **kwargs):
     """ Matrix Determinant
```

Comment:
The docstring still doesn't say anything about what the keyword arguments can be. 

For concrete matrices, the determinant takes a keyword argument 'method" which can be one of ... The default method is 'bareiss'.

Reply:
So, you are saying that I should update the docstring?
If so, could you suggest the format?
@normalhuman

Reply:
I suggested it in my previous comment.  

> Matrix determinant. If matexpr is a concrete matrix, the determinant takes a keyword argument 'method" which can be one of "bareiss", "berkowitz", and "lu"; the default is "bareiss".